### DISCUSSION TITLE
Can one instance use two addresses?

### DISCUSSION THREAD

**Discussion by cjstremick** [datalust/seq-tickets#1184] [Help] [answered]
I'm in the process of migrating to a new domain name and cannot do this is one step.  So I would like to let Seq receive events as the 'old' domain name and have the UI serve on the new one.  Then after migrating my various app configs to point at the new domain, I can retire the old one.

I can add the new address under api listenUris and change api canonicalUri to the new address (required to let oauth redirect properly).  I then changed the ssl thumbprint so the correct cert would be used for ssl:

`seq bind-ssl --thumbprint="[[ my new domain cert thumbprint]]"`

I think this is the problem, though.  This gets the UI working great on the new name, but event ingestion fails.  but I really need both certs to work.  One for the UI and still the old one for the event ingestion (suspecting this is why traffic is being dropped - because the cert is invalid).

Is this supported?  Can I just add two certs?

**Comment by cjstremick** [comment, marked as answer]
Nevermind.  I see that bind-ssl lets me specify a hostname.  This did the trick

**Reply by nblumhardt** [reply]
Thanks for the follow-up :+1:
